### PR TITLE
STRM-202: Add methods for stream access level in user and apikey APIs

### PR DESCRIPTION
**Pull request by dlazarenko-macrometa**
Added two methods `db.apiKeys().getStreamAccessLevel(keyId, stream)`, `db.user().getStreamAccessLevel(user, stream)` that need for validation access to a stream.
Added multi service connection via `.serviceHost(Service.C8STREAMS, "host", port)` in `C8DB`
Added `jwtUser("usernane")` in `C8DB`. It gives ability use users JWT instead of `service` users JWT

### CODE REVIEW

**Review by StoyanTc — APPROVED** [review]

File: src/test/java/com/c8db/C8ApiKeysTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
+/*
+ * DISCLAIMER
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
```
Comment (yehan-macrometa) [inline src/test/java/com/c8db/C8ApiKeysTest.java:8]: Shouldn't this be Macrometa copyright?

File: src/test/java/com/c8db/C8UserTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
+/*
+ * DISCLAIMER
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
```
Comment (yehan-macrometa) [inline src/test/java/com/c8db/C8UserTest.java:8]: Shouldn't this be Macrometa copyright?

File: src/main/java/com/c8db/internal/C8UserImpl.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2021 Macrometa Corp All rights reserved
```
Comment (yehan-macrometa) [inline src/main/java/com/c8db/internal/C8UserImpl.java:2]: Year: 2022

File: src/main/java/com/c8db/internal/InternalC8User.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021 Macrometa Corp All rights reserved.
```
Comment (yehan-macrometa) [inline src/main/java/com/c8db/internal/InternalC8User.java:2]: Year: 2022

**Review by yehan-macrometa — APPROVED** [review]
Added a few comments on copyright headers.
Other than that, this LGTM.

**Review by chandrashekhar-macrometa — APPROVED** [review]

File: src/test/java/com/c8db/C8ApiKeysTest.java (hunk 1 above)
Comment (dlazarenko-macrometa) [reply src/test/java/com/c8db/C8ApiKeysTest.java:8]: Changed

**Pull request merged**